scripts/customer-info/customer_info.yaml

```diff
@@ -0,0 +1,224 @@
+    - '  Middleman: 1'
+    - '  SourceSymlinkManifest: 1'
+    - '  GoLink: 1'
+    - '  GoCompilePkg: 1'
+    //java/...:
+    - '  GenProto: 1'
+    - '  TestRunner: 1'
+    - '  GenProtoDescriptorSet: 1'
+    - '  Action: 3'
+    - '  Middleman: 3'
+    - '  FileWrite: 3'
+    - '  TemplateExpand: 3'
+    - '  SymlinkTree: 3'
+    - '  SourceSymlinkManifest: 3'
+    - '  JavaDeployJar: 3'
+    - '  Turbine: 5'
+    - '  Javac: 8'
+    - '  JavaSourceJar: 10'
+    //kotlin/...:
+    - '  SymlinkTree: 1'
+    - '  Middleman: 1'
+    - '  SourceSymlinkManifest: 1'
+    - '  TemplateExpand: 1'
+    - '  KotlinFoldJarsRuntime: 1'
+    - '  JdepsMerge: 1'
+    - '  KotlinCompile: 1'
+    - '  JavaSourceJar: 1'
+    //scala/...:
+    - '  SymlinkTree: 1'
+    - '  Middleman: 1'
+    - '  TestRunner: 1'
+    - '  SourceSymlinkManifest: 1'
+    - '  TemplateExpand: 1'
+    - '  JavaIjar: 1'
+    - '  JavaSourceJar: 2'
+    - '  ScalaDeployJar: 2'
+    - '  Scalac: 2'
+    - '  FileWrite: 4'
+    //typescript/...:
+    - '  SymlinkTree: 1'
+    - '  Middleman: 1'
+    - '  TestRunner: 1'
+    - '  SourceSymlinkManifest: 1'
+    - '  FileWrite: 1'
+    - '  Genrule: 1'
+    - '  TsProject: 1'
+    - '  TsValidateOptions: 1'
+    - '  CopyFile: 3'
+  total_actions:
+  - 8
+  - 43
+  - 16
+  - 47
+  - 5
+  - 196
+  - 11
+  - 8
+- *id001
+- *id001
+- *id001
+- *id001
+- *id001
+- *id001
+- *id001
+bazel_targets:
+- - //kotlin:kotlin (2428b57)
+- - //docker/network:docker-network-test (d7452c8)
+  - //docker/network:jdk (2428b57)
+  - //docker/network:platform (2428b57)
+  - //docker/sandbox:dep0 (2428b57)
+  - //docker/sandbox:dep1 (2428b57)
+  - //docker/sandbox:dep2 (2428b57)
+  - //docker/sandbox:hello (2428b57)
+  - //docker/sandbox:hello-cc (2428b57)
+  - //docker/sandbox:hello-java (2428b57)
+- - //scala/com/engflow/example:exampleScala (2428b57)
+  - //scala/com/engflow/example:exampleScalaTest (d7452c8)
+- - //java/com/engflow/example:ExampleTest (d7452c8)
+  - //java/com/engflow/example:example (2428b57)
+  - //java/com/engflow/notificationqueue/demoserver:server (2428b57)
+  - //java/com/engflow/notificationqueue/demoserver:server_java_grpc_proto (2428b57)
+  - //java/com/engflow/notificationqueue/demoserver:server_java_proto (2428b57)
+  - //java/com/engflow/notificationqueue/demoserver:server_proto (2428b57)
+  - //java/com/engflow/notificationqueue:client (2428b57)
+  - //java/com/engflow/notificationqueue:engflowapis_java_proto (2428b57)
+  - //java/com/engflow/notificationqueue:eventstore_java_grpc (2428b57)
+  - //java/com/engflow/notificationqueue:notification_queue_java_grpc (2428b57)
+- - //go:go (2428b57)
+- - //csharp:csharp.exe (2428b57)
+- - //typescript:_validate_typescript_options (2428b57)
+  - //typescript:typescript (2428b57)
+  - //typescript:typesctript_test (d7452c8)
+  - //typescript:typesctript_test_0__deps (2428b57)
+- - //cpp:cpp (2428b57)
+relevant bazel flags and values:
+- &id002
+  - 2428b57:
```

Comment:
If we're consuming this elsewhere or a human is reading it, the configuration IDs won't mean much to us without access to the project. Could we use the target name/pattern here?

Reply:
This is challenging because multiple target name/patterns have the same config ID. So it would be like "//docker/sandbox:dep0, //docker/sandbox:dep1, ..." : "experimental_allow_runtime_deps_on_neverlink: 'true' experimental_limit_android_lint_to_android_constrained_java: 'false'"